src/views/Drawer/DrawerView.js

```diff
@@ -24,6 +24,7 @@ export type DrawerScene = {
 };
 
 export type DrawerViewConfig = {
+  drawerLockMode: 'unlocked' | 'locked-closed' | 'locked-open',
```

Comment:
Could we make this optional (`drawerLockMode?: ...`)? The DrawerLayout implementation defaults to unlocked anyways.